pages/linux/bluetoothd.md

```diff
@@ -0,0 +1,24 @@
+# bluetoothd
+
+> Daemon to manage bluetooth devices.
+> More information: <https://manned.org/bluetoothd>.
+
+- Start the daemon:
+
+`bluetoothd`
+
+- Start the daemon, logging to stdout:
+
+`bluetoothd --nodetach`
+
+- Start the daemon with a specific configuration file:
+
+`bluetoothd --configfile {{path/to/file}}`
```

Comment:
How about showing the default for a little variety and extra info?
```suggestion
`bluetoothd --configfile {{/etc/bluetooth/main.conf}}`
```

Reply:
@navarroaxel

Reply:
Check this: https://github.com/tldr-pages/tldr/blob/main/pages/linux/openfortivpn.md#L10

Reply:
I like that better.  I'll resolve this thread and start a new one with a suggestion.